Gemfile

```diff
@@ -1,24 +1,23 @@
-require 'rbconfig'
 source 'https://rubygems.org'
 gemspec
 
 group :development do
-  gem 'yard', '~> 0.9'
   gem 'rake', '~> 12.0'
+  gem 'standard', '~> 1.21'
+  gem 'webrick', '~> 1.7'
```

Comment:
Move `standard` and `webrick` dependency declaration over from the gemspec file to be grouped here with the other development dependencies.